packages/circuit-ui/components/Checkbox/Checkbox.tsx

```diff
@@ -241,6 +241,7 @@ export const Checkbox = forwardRef(
 
     return (
       <CheckboxWrapper className={className} style={style} noMargin={noMargin}>
+        {/* @ts-expect-error the noMargin prop is required */}
```

Comment:
We're not setting it because that would be a visual regression. Developers will need to pass it explicitly